packages/kolme-store/src/error.rs

```diff
@@ -20,14 +43,41 @@ pub enum KolmeStoreError {
     // kolme#144 - Reports a double insert (Block already exists with same hash and insert)
     #[error("Block already in database: {height}")]
     MatchingBlockAlreadyInserted { height: u64 },
+
     #[error("Transaction is already present in database: {txhash}")]
     TxAlreadyInDb { txhash: Sha256Hash },
+
+    #[error("get_height_for_tx: invalid height bytes in Fjall store for tx {txhash:?}, bytes: {bytes:?}, reason: {reason}")]
+    InvalidHeightInFjall {
+        txhash: Sha256Hash,
+        bytes: Vec<u8>,
+        reason: String,
+    },
+
     #[error("{0}")]
     Other(String),
 }
 
 impl KolmeStoreError {
     pub fn custom<E: std::error::Error + Send + Sync + 'static>(e: E) -> Self {
-        Self::Custom(Box::new(e))
+        Self::Custom(e.to_string())
+    }
+}
+
+impl From<fjall::Error> for KolmeStoreError {
+    fn from(e: fjall::Error) -> Self {
+        KolmeStoreError::Other(format!("Fjall error: {e}"))
```

Comment:
Can we have a variant for `KolmeStoreError::Fjall` instead so we don't have to convert the errors to Strings? I know I only left the string comment in one place previously, but it applies to all of the error data, the goal is to make error messages more easily inspectable.